internal/venice-common/src/main/java/com/linkedin/venice/LogMessages.java

```diff
@@ -0,0 +1,7 @@
+package com.linkedin.venice;
+
+public class LogMessages {
+  // Replica with ERROR status with the following message implies the job was killed instead of actual ingestion error
```

Comment:
Sorry for the nitpick... but can we say that this string is used in server-to-controller communication and that it should not be changed, otherwise a cluster running with mixed versions might not behave properly? Also, let's please use JavaDoc `/** */` instead of `//` so that it populates the JavaDoc and tool tips properly...

Reply:
Sure, updated the javadoc